libres/tests/enkf/test_enkf_fs.cpp

```diff
@@ -151,3 +155,30 @@ TEST_CASE("block_fs", "[enkf_fs]") {
         block_fs_close(bfs);
     }
 }
+
+
+void mount(void * args)
+{
+    enkf_fs_type *fs2 = enkf_fs_mount("mnt");
+    enkf_fs_decref(fs2);
+}
+
+
+TEST_CASE("Mount filesystem read/write twice", "[enkf_fs]") {
+    enkf_fs_create_fs("mnt", BLOCK_FS_DRIVER_ID, false);
```

Comment:
This should probably use the `WITH_TMPDIR;` RAII statement so that the `mnt` directory is created in an appropriate temporary directory. See https://github.com/equinor/ert/blob/5021243874589b99502cc8295d27c627adf87e64/libres/tests/analysis/test_save_parameters.cpp#L51